### ISSUE TITLE
Question about <SamplingPeriod>

### ISSUE THREAD

**Issue by rdiez** [intel/thermal_daemon#418]
I couldn't find much documentation about `<SamplingPeriod>` under `<CoolingDevice>`, in configuration file thermal-conf.xml .

Are the units in seconds? They could be milliseconds. I tried with values 1 and 5, and it made no noticeable difference.

Can you specify a floating point value like 0.5 ?

Do very low values make sense?

**Comment by spandruvada** [comment]
You can't specify in decimal point. They are seconds. This is the time, that device has noticeable affect after you change state.

**Comment by rdiez** [comment]
First of all, thanks for your answer.

> This is the time, that device has noticeable affect after you change state.

I do not understand what you mean. Could you rephrase that sentence?
